release.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# Perform a docker-machine release
+# GITHUB_TOKEN=XXXXX ./release.sh 0.X.X
+
+#GITHUB_USER=docker
+GITHUB_USER=jeanlaurent
+GITHUB_REPO=machine
+
+
+function display {
+    echo "🐳   $1"
+    echo ""
+}
+
+function checkError {
+    if [ "$?" -ne 0 ]; then
+        echo "😡   $1"
+        exit 1
+    fi
+}
+
+VERSION=$1
+GITHUB_VERSION="v$1"
+PROJECT_URL="git@github.com:$GITHUB_USER/$GITHUB_REPO"
+RELEASE_DIR=../release-$VERSION
+DOCKER_MACHINE=$GOPATH/src/github.com/docker/machine/bin/docker-machine
```

Comment:
Mine are usually in `bin/docker-machine` or `/usr/local/bin/docker-machine`-- why not simply refer to `docker-machine` directly, or just do one check at the beginning like `which docker-machine`?